src/lib/Components/StickyTabPage/StickyTabScrollView.tsx

```diff
@@ -13,6 +14,8 @@ export const useStickyTabContext = () => {
   return useContext(StickyTabScrollViewContext)
 }
 
+const AnimatedFlatList: typeof FlatList = Animated.createAnimatedComponent(FlatList)
```

Comment:
Oh nice, is this a common pattern?

Reply:
Yep. I'm unsure how it works under the hood but it's supported behaviour for the core RN components at least.